src/main/java/seedu/address/logic/commands/UpdateEventCommand.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REPEATABLE_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REPEATABLE_INTERVAL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_UPDATE_INDEX;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.repeatable.Event;
+
+/**
+ * Updates an event inside a project.
+ */
+public class UpdateEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "updateE";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates an event of a project specified "
```

Comment:
This message can be improved, please refer to deleteE command and standardise the formatting with that.